test/test_spectral_ops.py

```diff
@@ -874,6 +874,22 @@ def plan_cache_max_size(device, n):
                             self.assertEqual(torch.backends.cuda.cufft_plan_cache.max_size, 10)  # default is cuda:0
                         self.assertEqual(torch.backends.cuda.cufft_plan_cache.max_size, 11)  # default is cuda:1
 
+    @deviceCountAtLeast(1)
```

Comment:
This is not needed, is it? (i.e. `@onlyCUDA` decorator already implies that you have a device to run the test) 
```suggestion
```